src/strpkg/merge.nim

```diff
@@ -79,7 +92,10 @@ proc merge_main*() =
       targets = extracted.targets
     else:
       if not extracted.targets.same(targets):
-        quit &"[strling] Error: inconsistent bam header for {binfile}. Were all samples run on the same reference genome?"
+        if allow_diff_chroms:
```

Comment:
i think we can remove this. just use: 
```
if not allow_diff_chroms:
    quit ...
```
we dont need `same_lengths` to adjust targets (but it could be used to check that chroms with same names have same lengths.